src/Views/Solvers/WordSearch.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import WordSearchComponent from '../../Solvers/WordSearchComponent';
+import './WordSearch.css';
+
+class WordSearch extends Component {
+  public render() {
+    return (
+      <div className="Search">
+        <div className="helpText">
+          <p>
+            Enter the list of words to find (one per line), and the grid to find words in

```

Comment:
nit: wording

@beckbria Any suggestions?

Reply:
Any sentence trying to explain both boxes seems destined to become ambiguous.  Can we make a label over each text box?  Potential phrasings:

Enter the list of words to find, one word per line
Enter the word search grid, one row of letters per line